File: TestBankApp/app/src/main/java/com/example/bankapp/remote/LoginAPI.java
package com.example.bankapp.remote;

import com.example.bankapp.model.user.UserAccountModel;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;

public interface LoginAPI {
    @FormUrlEncoded
    @POST("login")
    Call<UserAccountModel> login(
            @Field("user") String userName,
            @Field("password") String password);
}

File: app/src/main/java/com/br/rafael/TesteAndroidv2/data/api/LoginApi.kt
package com.br.rafael.TesteAndroidv2.data.api

import retrofit2.Call
import com.br.rafael.TesteAndroidv2.data.model.LoginResponse
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface LoginApi {

    @FormUrlEncoded
    @POST("login")
    fun userLogin(@Field("user") user: String, @Field("password") password: String): Call<LoginResponse>
}